charts/fluentd/templates/fluentd-configmap.yaml

```diff
@@ -56,6 +56,14 @@ data:
       max_bytes 500000
       max_lines 1000
     </match>
+    # Concat log truncated by docker 16KB limit, single line JSON, https://github.com/fluent-plugins-nursery/fluent-plugin-concat
```

Comment:
do we have this pluging in our fluentd docker image? 

nvm already found it https://github.com/astronomer/ap-vendor/blob/main/fluentd/include/Gemfile#L9